pkg/kubectl/cmd/apiversions.go

```diff
@@ -33,15 +32,15 @@ func NewCmdApiVersions(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"apiversions"},
 		Short:   "Print available API versions.",
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunApiVersions(f, out)
+			err := RunApiVersions(f, out, cmd)
 			cmdutil.CheckErr(err)
 		},
 	}
 	return cmd
 }
 
-func RunApiVersions(f *cmdutil.Factory, out io.Writer) error {
-	if os.Args[1] == "apiversions" {
+func RunApiVersions(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
+	if calledByAlias(cmd) {
```

Comment:
I'd be more inclined to take this if it were a simple adding of ```len(os.Args) > 1 && ``` in the appropriate places. That stops crashes while not changing any semantics.